editor/src/components/editor/import-wizard/import-wizard.tsx

```diff
@@ -197,21 +228,44 @@ function ActionButtons({ importResult }: { importResult: ImportOperationResult |
       </React.Fragment>
     )
   }
-  if (importResult == ImportOperationResult.Warn) {
+  if (importResult.result == ImportOperationResult.Warn) {
     return (
       <React.Fragment>
         <div style={textStyle}>Project Imported With Warnings</div>
         <Button onClick={hideWizard} style={buttonStyle}>
-          Continue
+          Continue To Editor
         </Button>
       </React.Fragment>
     )
   }
-  if (importResult == ImportOperationResult.Error) {
+  if (
+    importResult.result == ImportOperationResult.Error ||
+    importResult.result == ImportOperationResult.CriticalError
+  ) {
     return (
       <React.Fragment>
-        <div style={textStyle}>Error Importing Project</div>
-        <Button style={buttonStyle}>Import A Different Project</Button>
+        <div style={textStyle}>
```

Comment:
if we're in an error mode (but no critical error), show a button for `Continue Anyway`